osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableStrongHandler.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Taiko.Judgements;
+
+namespace osu.Game.Rulesets.Taiko.Objects.Drawables
+{
+    /// <summary>
+    /// Used as a nested hitobject to provide <see cref="TaikoStrongJudgement"/>s for <see cref="DrawableTaikoHitObject"/>s.
+    /// </summary>
+    public abstract class DrawableStrongHandler : DrawableTaikoHitObject
```

Comment:
`DrawableStrongNestedHit` maybe? not sure about using "handler" here since we already use it in so many other places.